unit-tests/src/test/scala/scala/scalanative/posix/TimeTest.scala

```diff
@@ -38,6 +39,9 @@ class TimeTest {
   }
 
   @Test def localtimeShouldTransformTheEpochToLocaltime(): Unit = {
+    assumeFalse(
+      "Skipping localtime test since FreeBSD hasn't the 'timezone' variable",
+      System.getProperty("os.name").equals("FreeBSD"))
```

Comment:
Rather than repeating this in the tests, I think we can add it to `Platform.scala` in the test `utils` directory. 

Maybe `isFreeBSD`?.

Just out of curiosity is this only FreeBSD or could we expect the other variants to work as well?

Reply:
You can expect it for other variants as well (OpenBSD, etc.)